include/usearch/index_plugins.hpp

```diff
@@ -50,8 +50,10 @@
 #define SIMSIMD_NATIVE_F16 !USEARCH_USE_FP16LIB
 
 #if !defined(SIMSIMD_TARGET_X86_AVX512) && defined(USEARCH_DEFINED_LINUX)
+#if defined(__AVX512F__) && defined(__AVX512FP16__) && defined(__AVX512VNNI__) && defined(__AVX512VPOPCNTDQ__)
 #define SIMSIMD_TARGET_X86_AVX512 1
 #endif
+#endif
```

Comment:
These should be patched in SimSIMD, not USearch, I believe. The USearch package shouldn't know exactly which subsets of AVX512 are used by SimSIMD.